noir-projects/aztec-nr/aztec/src/context/private_context.nr

```diff
@@ -401,7 +401,7 @@ impl PrivateContext {
         args: [Field; ARGS_COUNT]
     ) {
         let args_hash = hash_args_array(args);
-        assert(args_hash == arguments::pack_arguments(args));
+        assert(args_hash == arguments::pack_arguments_array(args));
```

Comment:
does this mean it should be a breaking change? if so can you add an `!` in your pR title:

`feat!: contract interafcaes....`

Reply:
Technically it is, but I don't think this oracle is ever called outside of the contexts...I'll add it anyway